settings.py

```diff
@@ -10,7 +10,16 @@
     'pgtrigger',
 ]
 # Database url comes from the DATABASE_URL env var
-DATABASES = {'default': dj_database_url.config()}
+DATABASES = {
```

Comment:
`dj_database_url` is convenient. If you want to override the URL you can specify the `default=` argument to `config` call like this:

```
dj_database_url.config(default="postgres://postgres:postgres@localhost:5432/postgres")
```

Reply:
I did try this
```
DATABASES = {'default': dj_database_url.config(default="postgres://postgres:postgres@localhost:5432/postgres")}
```
but it always resulted in the error
```
E       django.db.utils.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
E       	Is the server running on that host and accepting TCP/IP connections?
E       connection to server at "localhost" (::1), port 5432 failed: Address not available
E       	Is the server running on that host and accepting TCP/IP connections?
```

Reply:
The correct place to fix this is in the docker compose file as I've commented above

Reply:
Nice! That worked :)